README.md

```diff
@@ -31,35 +35,43 @@ support basic PCM formats, as well as built-in decoders for FLAC (dr_flac) and
 MP3 (minimp3). Application-defined decoders are also supported in case the
 default set are insufficient.
 
-And much more...
+And much more…
+
+Installation
+------------
+As of the moment, Alure 2 is only packaged for NixOS, on which it can be
+installed by
+
+    nix-env -iA nixos.alure2
+
+For other operating systems, please see follow the next section to build from
+the source code.
```

Comment:
I'm not sure it's a good idea to mention which specific OSs and distros have an Alure2 package since new ones can pop up at any time and some may become outdated.

Reply:
Sure I will remove this part then.